server/db/migrations/20220922101628_produce.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('produce', (table) => {
+    table.increments('id').primary()
+    table.string('name')
+    table.string('display_name')
```

Comment:
why do you have name and display_name? in your Readme you also dont include `display_name` in your table example

Reply:
after reviewing the whole thing I see why you have it. but the second part of the comment still applies

Reply:
The duplication here is to support the external API. Results can vary with a query built using the contents of the `display_name`

Edamam will get results for sweet potato recipes with a query of `Kūmara`. While a search for `Cauliflower/Kareparāoa` will produce zero results. This issue persists for most `produce` where they have the English/Maori name format.

This data is expected to be used to: display in-season produce to the user and generate a request query for Edamam based on user input.

I have updated the README on the `dev` branch